aws/resource_aws_db_instance_test.go

```diff
@@ -4349,3 +4375,44 @@ resource "aws_db_instance" "test" {
 }
 `, rName, rName, rName, rName)
 }
+
+func testAccAWSDBInstancePerformanceInsights(n int) string {
```

Comment:
This change isn't possible because when insights are disabled, the kms key and retention values have to be empty. So I've used different test configuration to test disabled insights.